tests/unit/test_register_handlers.py

```diff
@@ -616,6 +634,23 @@ async def test_register_basic_happy_path_en(self):
         await self.post_register_child_dob(display_region)
         await self.post_register_person_summary(display_region)
 
+    @unittest_run_loop
+    async def test_register_rhsvc_submission_error_400_en(self):
+        display_region = 'en'
```

Comment:
I'm confused about this test. the method name suggests testing 400 errors, but the calls test for 200.

Reply:
It's testing what happens when RHSvc returns a 400 response for the new case call, but it still has to successfully step through the previous steps

Reply:
OK I understand